lib/bridge/MatrixHandler.js

```diff
@@ -451,6 +451,13 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
         return;
     }
+    else if (cmd === "!quit") {
+        clientList.filter(
+            (bridgedClient) => bridgedClient.server.domain === ircServer.domain
+        ).forEach((bridgedClient) => {
+            bridgedClient.kill();
+        });
```

Comment:
They will see their connection drop and come back again. I could add even more feedback but it would feel redundant.